chef/cookbooks/provisioner/recipes/setup_base_images.rb

```diff
@@ -446,13 +446,15 @@
 node.set[:provisioner][:repositories] = Mash.new
 node.set[:provisioner][:available_oses] = Mash.new
 
+ntp_servers = search(:node, "roles:ntp-server")
+ntp_servers_ips = ntp_servers.map { |n| Chef::Recipe::Barclamp::Inventory.get_network_by_type(n, "admin").address }
```

Comment:
Metrics/LineLength: Line is too long. [115/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)